### PR TITLE
feat: Add API v2 finding samples

### PR DESCRIPTION
**Pull request by rafaelMurata**
## Description

Fixes #https://buganizer.corp.google.com/issues/333521328

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md)
- [ ] `pom.xml` parent set to latest `shared-configuration`
- [ ] Appropriate changes to README are included in PR
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample 
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  public static void main(String[] args) throws IOException {
+    // TODO(Developer): Replace the sample resource name
+    // organizationId: Google Cloud Organization id.
+    String organizationId = "{google-cloud-organization-id}";
+
+    createSource(organizationId);
+  }
+
+  public static Source createSource(String organizationId) throws IOException {
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java:37]: Add a method comment here.

File: security-command-center/snippets/src/test/java/vtwo/FindingsIT.java [hunk 2]
```diff
@@ -114,20 +120,32 @@ public void testListAllFindings() throws IOException {
...
+
     assertThat(stdOut.toString()).contains("count: 1");
   }
+
+  @Test
+  public void testSetFindingsByStateInactive() throws IOException {
+    SetFindingsByState.setFindingState(ORGANIZATION_ID, LOCATION, SOURCE.getName().split("/")[3],
+        FINDING_1.getName().split("/")[7]);
+
+    assertThat(stdOut.toString()).contains("Updated Finding: ");
```
Comment (Sita04) [inline security-command-center/snippets/src/test/java/vtwo/FindingsIT.java:148]: Change this to assert the state in the returned response from `SetFindingsByState.setFindingState`.

**Review by Sita04 — APPROVED** [review]
Thanks for the samples. LGTM.
Kindly address the couple comments before merging.

File: security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java (hunk 1 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java:49]: done

File: security-command-center/snippets/src/test/java/vtwo/FindingsIT.java (hunk 2 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/test/java/vtwo/FindingsIT.java]: done

File: security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+   * @param organizationId the identifier of the GCP organization where the finding is reported
+   * @param location       the geographical location or data center where the finding is associated
+   * @param findingId      a unique identifier for the finding within the specified source
+   * @param sourceId       the identifier of the source where the finding was detected, such as a
+   *                       scanner or other security tool
+   * @param category       an Optional describing the category of the finding, which could include
+   *                       classifications like 'misconfiguration', 'malware', etc.
+   * @return Finding the newly created finding object that contains detailed information about the
+   * security issue detected
+   * @throws IOException if there is an error in communication with the Security Command Center API
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java:73]:
```suggestion
```

File: security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+   * Creates a new "source" in the Security Command Center.
+   * <p>
+   * This method sets up and creates a source in SCC, which can be used to group and identify
+   * specific security findings generated by third-party scanners or processes. A source acts as a
+   * logical container in SCC for security findings, allowing different tools and services to report
+   * identified issues under the same organizational category.
+   *
+   * @param organizationId the identifier of the GCP organization where the source will be created
+   * @return Source the source created in the Security Command Center
+   * @throws IOException if an input/output error occurs during source creation
```
Comment (Sita04) [inline security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java:47]:
```suggestion
```

**Review by Sita04 — CHANGES_REQUESTED** [review]

**Comment by Sita04** [comment]
@rafaelMurata PTAL

**Comment by rafaelMurata** [comment]
@Sita04 
done

File: security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java (hunk 3 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java:71]: done

File: security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java (hunk 4 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java:46]: done

**Comment by rafaelMurata** [comment]
gcbrun

File: security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java (hunk 3 above)
Comment (Sita04) [reply security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java:71]: @rafaelMurata Retain only the first line of the comment and remove the ones highlighted here.

File: security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java (hunk 4 above)
Comment (Sita04) [reply security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java:46]: @rafaelMurata Retain only the first line of the comment and remove the ones highlighted here.

File: security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java (hunk 3 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/main/java/vtwo/findings/CreateFindings.java]: done

File: security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java (hunk 4 above)
Comment (rafaelMurata) [reply security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java]: done

**Review by Sita04 — APPROVED** [review]

**Comment by rafaelMurata** [comment]
@Sita04 
please run kokoro again. Thanks.

**Pull request merged**